packages/pilot/src/containers/AddRecipient/ConfigurationStep/Anticipation/index.js

```diff
@@ -62,27 +70,108 @@ const renderAnticipationInput = (data, t) => {
     equals('manual')
   )
 
-  let inputProps = {
-    label: t('pages.add_recipient.anticipation_days'),
-    name: 'anticipationDays',
-  }
+  let inputProps
 
-  if (volumePercentage(anticipationModel)) {
+  if (volumePercentage(anticipationModel) || anticipationModel === 'custom') {
```

Comment:
Esta função `volumePercentage` havia me passado batido, ela pode ser escrita de uma maneira mais simples de ler ex:
```suggestion
  if (isVolumePercentage(anticipationModel) || anticipationModel === 'custom') {
```

Reply:
Atualizado. Faz sentido esse nome :+1: